src/modules/commentCounter.test.js

```diff
@@ -0,0 +1,20 @@
+const { countComments } = require('./commentCounter.js');
+
+describe('Count comments of a meal item', () => {
+  test('No comments', () => {
+    const arr = [];
+    expect(countComments(arr)).toBe(0);
+  });
+  test('Expect to get 2 comments back', () => {
+    const comments = [{
+      creation_date: '2022-10-26',
+      username: 'kenny',
```

Comment:
- [ ] When a user clicks on the "Comment" button, the data is expected to get recorded in the involvement API and the Screen is updated. For now your comment will update the screen after reloading the page. Kindly fix it.
![Screenshot from 2022-10-29 15-11-21](https://user-images.githubusercontent.com/65860678/198833850-eeeaafa9-5b92-4000-83b6-297a101cccd5.png)

Reply:
Hello, good day, the comment will be seen after the user clicks on the comment button right after adding a new comment

Reply:
That will be right. For now when I click on comment, the comment will be added but not update the screen. Just confirm and ask for an other review.

Reply:
Okay, I have already implemented the comment to display on page load too